sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -121,18 +121,23 @@ should be used.
 
 There is experimental support for SYCL for CUDA devices.
 
-To enable support for CUDA devices, the following arguments need to be added to 
-the CMake command when building the SYCL compiler.
+To enable support for CUDA devices, follow the instructions for the Linux
+SYCL toolchain, but replace the cmake line with the following one:
```

Comment:
replace the cmake command with the following one?

Reply:
Addressed in 7d027efeda7845a8d6df6f7fa109d4c73451ea61